src/ACEtk/details/verify/Sorted/src/ctor.hpp

```diff
@@ -0,0 +1,24 @@
+template< typename Arg = std::initializer_list< typename Range::value_type >,
+          utility::Require< false, IsSorted, Arg > = true,
+          utility::Require< true, utility::is_range, Arg > = true,
+          utility::Require< true, std::is_constructible, Range, Arg > = true
+        >
+Sorted( Arg&& arg ) :
+  Range( check( std::forward< Arg >( arg ) ) )
+{ }
+
+template< typename... Args, 
+          utility::Require< true, std::is_constructible, Range, Args... > = true
+        >
+Sorted( Args&&... args ):
+  Range( check( std::forward< Args >( args )... ) )
```

Comment:
This doesn't make sense. Check cannot be applied to multiple arguments. You can only verify sorted-ness after constructing the parent. See the lazy verification in the example I sent.

Reply:
Aside from the coverage, I think this is the last thing to be addressed.